CODEOWNERS

```diff
@@ -20,3 +20,8 @@ requirements-training.txt @microsoft/onnxruntime-admin
 requirements.txt.in @microsoft/onnxruntime-admin
 /cgmanifests/cgmanifest.json @microsoft/onnxruntime-admin
 /cgmanifests/generated/cgmanifest.json @microsoft/onnxruntime-admin
+
+# Public facing APIs
+/onnxruntime/include/onnxruntime/core/session/onnxruntime_c_api.h @microsoft/onnxruntime-api
```

Comment:
How about Python and C# ?

Reply:
yeah, was debating if they should be added as they don't necessarily break ABI. Now I think I'll just add them for completeness and tighter control.